appinventor/buildserver/src/com/google/appinventor/buildserver/tasks/android/RunApkSigner.java

```diff
@@ -33,12 +33,15 @@ public TaskResult execute(AndroidCompilerContext context) {
         "-ks", context.getKeystoreFilePath(),
         "-ks-key-alias", "AndroidKey",
         "-ks-pass", "pass:android",
+        // "-key-pass", "pass:android",
```

Comment:
Is this needed?

Reply:
I agree with disabling stdin. I'm not sure it's worth keeping these around as we also have the discussion here in the PR about why this was decided.